src/lib/i18n.js

```diff
@@ -7,6 +7,10 @@ try {
 
 const language = __LANGUAGE__ || 'en-US'; // eslint-disable-line
 
+const getLocale = lang => {
+    return lang.substr(0, lang.indexOf('-'));
```

Comment:
Do we need to handle the case where the lang doesn't have a dash?

Reply:
I believe all of our supported langs have a dash.